substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/util/LLVMTargetSpecific.java

```diff
@@ -158,7 +158,12 @@ public int getFramePointerDwarfRegNum() {
 
             @Override
             public List<String> getLLCAdditionalOptions() {
-                return Collections.singletonList("-no-x86-call-frame-opt");
+                List<String> list = new ArrayList<>();
+                list.add("-no-x86-call-frame-opt");
```

Comment:
Would be good to say why we need this?

Reply:
"-no-x86-call-frame-opt" is added [here](https://github.com/oracle/graal/commit/a30a2bd7e5a013684b5ecf0ec01a5644488c7e6f#diff-dadb8fe199752800b0a1803c69a3c12e6933ecac66c66a5ba85115e622ed6838)

Reply:
This `-no-x86-call-frame-opt` option was already there, I just added the `-mtriple` option if it's targeting IOS, by copying what we've already done [here](https://github.com/oracle/graal/blob/f5f20e987cb32de902eb68b2589837ada74bc069/substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/util/LLVMTargetSpecific.java#L233-L235)